apps/meteor/server/services/video-conference/service.ts

```diff
@@ -1034,4 +1070,124 @@ export class VideoConfService extends ServiceClassInternal implements IVideoConf
 		await this.runVideoConferenceChangedEvent(call._id);
 		await this.sendAllPushNotifications(call._id);
 	}
+
+	private async isPersistentChatEnabled(): Promise<boolean> {
```

Comment:
Does this need to be async?
```suggestion
	private isPersistentChatEnabled(): boolean {
```

Reply:
It did initially cause I was checking for more things inside of it but it doesn't need to be async anymore with the current code. I'll change it.